docs/Installing.asciidoc

```diff
@@ -707,6 +707,33 @@ This will allow the workers to download the assets from the web UI and use them
 locally. If `TESTPOOLSERVER` is set tests and needles will also be cached by the
 worker.
 
+=== Enable linking files referred by job settings
+
+Specific job settings might refer to files within the test distribution.
+You can configure openQA to display links to these files within the job settings tab.
+To enable particular settings to be presented as a link within the settings tab
+one can setup the relevant keys in `/etc/openqa/openqa.ini`.
+
+[source,ini]
+--------------------------------------------------------------------------------
+# Configuration for Setting UI
+[job_settings_ui]
+keys_to_render_as_links=FOO,AUTOYAST
+--------------------------------------------------------------------------------
+
+The keys should be defined in the `etc/openqa/settings_ui_links.ini` with assigned directories
+that the OpenQA settings page should render as links.
+
+[source,ini]
+--------------------------------------------------------------------------------
+# Definition of key patterns for Setting UI
+[links]
+FOO=bar,autoyast_sle15
+--------------------------------------------------------------------------------
+
+Those directories should be located in two places. Either under `data` folder or
+under the root of the `CASEDIR` path.
```

Comment:
As the "data folder" has no meaning for openQA itself but was a convention only within os-autoinst-distri-opensuse this should be explained further, e.g. just the following should be enough: "Either under the root of the `CASEDIR` path or a `data` folder within `CASEDIR`"

Reply:
Hmm, I'm not sure about that, we have `data` hardcoded in testapi, so it's definitely not `os-autoinst-distri-opensuse` specific, rather `os-autoinst` specific.

Reply:
oh, yes. you are right. So to be more correct: "convention within os-autoinst and the test distributions, not openQA itself". Well, anyway my suggestion does not conflict with that I guess :)

Even more so this is why I suggested in https://github.com/os-autoinst/openQA/pull/3203#discussion_r452655525 that we do not need any complicated config file changes at all which I would still suggest to follow.

Reply:
modified as suggested